src/Plugin/ultimate_cron/Launcher/SerialLauncher.php

```diff
@@ -209,28 +209,16 @@ public function launch($job) {
       return FALSE;
     }
 
-    if ($this->currentThread) {
-      $init_message = t('Launched in thread @current_thread', array(
-        '@current_thread' => $this->currentThread,
-      ));
-    }
-    else {
-      $init_message = t('Launched manually');
-    }
-    $log_entry = $job->startLog($lock_id, $init_message);
-
-    drupal_set_message(t('@name: @init_message', array(
-      '@name' => $job->id(),
-      '@init_message' => $init_message,
-    )));
+    $log_entry = $job->startLog($lock_id);
```

Comment:
Still not convinced about touching things like startLog(), those don't end up in the watchdog but a separate table and we afaik create an entry anyway, so it doesn't hurt to keep this.